### ISSUE TITLE
-peek-at cursors

### ISSUE THREAD

**Issue by mhuebert** [reagent-project/reagent#157]
I'm working on a project in which sometimes I want to dereference a cursor inside a component without triggering a reaction (eg. I want to show the value but only re-render on some other event).

I found [-peek-at in ratom.cljs](https://github.com/reagent-project/reagent/blob/master/src/reagent/ratom.cljs#L257)  and saw that it does exactly what I need, so I put IComputedImpl in RCursor and copied over the same implementation as was in Reaction. It appears to work exactly as intended (I'm fairly new to protocols and types in Clojure so I wasn't sure the best way to make the change). 

Is this something that might be added to master or is there a reason not to provide -peek-at ability to cursors?

**Comment by mike-thompson-day8** [comment]
@mhuebert  Adding this would fine.  I too have had situations where this would be useful.

**Comment by holmsand** [comment]
Yes, that might be a good idea. In the meantime, you can use something like 

```clj
(swap! some-atom identity)
```

**Comment by mike-thompson-day8** [comment]
@holmsand  I don't understand this `swap!` idea.  

As I understand it, @mhuebert wishes he could use `(peek-at ratom)` to get the value out of the ratom without having that ratom captured.  `(deref ratom)` does the same but causes the ratom to be captured.

**Comment by holmsand** [comment]
swap! returns the value produced by identity, which will be the original value of the ratom - and doesn't capture a deref.
